src/components/WwwFrame/LendMenu/LendListMenu.vue

```diff
@@ -150,7 +150,7 @@
 						</li>
 						<li>
 							<a
-								href="/lend/countries-not-lent"
+								href="/lend-category-beta?countries-not-lent-to=true"
```

Comment:
For this experiment, we can just change the lend menu, so this component and `LendMegaMenu.vue`.

And we only want to use the new URL if the experiment is enabled.